mindocr/losses/det_loss.py

```diff
@@ -155,6 +155,7 @@ def construct(self, pred: Tensor, gt: Tensor, mask: Tensor) -> Tensor:
         pos_count = positive.sum(axis=(1, 2), keepdims=True).astype(ms.int32)
         neg_count = negative.sum(axis=(1, 2), keepdims=True).astype(ms.int32)
         neg_count = ops.minimum(neg_count, pos_count * self._negative_ratio).squeeze(axis=(1, 2))
+        neg_count = ops.maximum(neg_count, 10)  # in case when an image has no text instances
```

Comment:
any special meaning of 10?

Reply:
No meaning, just to penalize pixels with 10 highest errors when an image has no positive samples (no text instances). Shouldn't affect the training process since all (or most of) the images must have text polygons in them.

The original work set loss to 0 for images with no text instances, i.e. simply skipped them.

Reply:
get it , better to add a comment in code of why adding 10, others may be confused